app/Http/Controllers/Api/V1/CountryController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Http\Controllers\Api\V1;
+
+use App\Traits\ApiResponser;
+use OpenApi\Attributes as OA;
+use App\Services\CountryService;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Country\Index;
+use App\Http\Resources\Country\Collection as CountryCollection;
+
+class CountryController extends Controller
+{
+    use ApiResponser;
+
+    public function __construct(private CountryService $countryService)
+    {
```

Comment:
@dhrumin-7span Make it same as UserStatusChangeController

Reply:
Changed according to UserStatusController.